CHANGELOG.md

```diff
@@ -381,7 +381,7 @@ Note that this project **does not** adhere to [Semantic Versioning](http://semve
 - We moved the union/intersection view button in the group sidepane to the left of the other controls. [#8202](https://github.com/JabRef/jabref/pull/8202)
 - We improved the Drag and Drop behavior in the "Customize Entry Types" Dialog [#6338](https://github.com/JabRef/jabref/issues/6338)
 - When determining the URL of an ArXiV eprint, the URL now points to the version [#8149](https://github.com/JabRef/jabref/pull/8149)
-- We Included all standard fields with citation key when exporting to Old OpenOffice/LibreOffice Calc Format [#8176](https://github.com/JabRef/jabref/pull/8176)
+- We Included all standard fields with citation key when exporting to Old OpenOffice/LibreExpoa Calc Format [#8176](https://github.com/JabRef/jabref/pull/8176)
```

Comment:
@Siedlerchr What is "LibreExpoa"? Is this change in CHANGELOG.md really intended?

Reply:
I was editing this on the github, this was not intended

Reply:
Fixed it directly in `main` at https://github.com/JabRef/jabref/commit/461e915105dfb5acd53b917b541a8a93026b8dad